js/downloads.js

```diff
@@ -22,7 +22,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 // 2.4.0+
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, sources];
 
-addRelease("2.4.0", new Date("11/02/2018"), packagesV9, true);
+addRelease("2.4.1", new Date("04/01/2019"), packagesV9, true);
```

Comment:
Ur, it should be March 31.
- https://lists.apache.org/thread.html/fbec33f7e601f25b8c950ec42a3ee287bd8623d973ea3a2c8d7f2fbb@%3Cdev.spark.apache.org%3E